specs/order_spec.rb

```diff
@@ -78,33 +80,44 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+      result = Grocery::Order.all
+      result.must_equal (CSV.read("/support/orders.csv"))
```

Comment:
`Order.all` should return an array of `Order` instances **not** an array of arrays!